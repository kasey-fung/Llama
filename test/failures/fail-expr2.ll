int add(int a, int b):
    return a + b

int main():
    int x
    char y

    y = 'a'

    x = add(1, 2)
    y * x
    print(x)


