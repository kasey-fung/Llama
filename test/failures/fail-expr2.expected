Fatal error: exception Failure("illegal binary operator char * int in y * x")